test/IntegrationTests/GraphQLTests.cs

```diff
@@ -73,6 +73,7 @@ public GraphQLTests(ITestOutputHelper output)
     public async Task SubmitsTraces()
     {
         SetEnvironmentVariable("OTEL_SERVICE_NAME", ServiceName);
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_GRAPHQL_SET_DOCUMENT", bool.TrueString);
```

Comment:
As it is our instrumentation, can we execute test twice with both values for the env. variable?

Reply:
I was just between doing it now or doing it later after tests are stabilized. I don't have a strong opinion on it, so that's why the open question.

Reply:
well added the missing set... let's see how it behaves.